src/functions-reference/higher-order_functions.Rmd

```diff
@@ -311,6 +320,32 @@ Because these are all `data` arguments, they must be defined in either the data
 or transformed data blocks. They cannot be parameters, transformed parameters
 or functions of parameters or transformed parameters.
 
+### Arguments to the adjoint ODE solver
+
+The arguments to the adjoint ODE solver are the same for specification
+of the basic ODE problem. The numerical tuning parameters provided
+include a greater set as the adjoint ODE solver is numerically more
+complex. The adjoint ODE solver runs a forward ODE solve followed by a
+backward ODE solve. During the backward ODE solve, the forward
+solution is approximated by a polynomial with support points of
+controllable density. Along with the backward ODE solve one quadrature
+problem per parameter is solved. The different phases can be
+controlled with the arguments:
```

Comment:
Oh actually when I rendered the docs I see how this is done (so that all the arguments wouldn't be specified). Is there a way to section this up differently so that all the arguments to the adjoint function are specified beside the signature? It's annoying to have to look around for things. If the answer is no, not really, then I guess that's the answer.

Reply:
I just added the arguments once more... for a ref doc this is fine, I think.